tests/python/rest_api/test_quality_control.py

```diff
@@ -1244,3 +1244,38 @@ def test_accumulation_annotation_conflicts_multiple_jobs(self, admin_user):
             )
 
         assert task_confusion_matrix == [[2, 0, 1], [0, 0, 0], [1, 0, 0]]
+
+    @pytest.mark.parametrize("task_id", [8])
+    def test_can_compute_quality_if_non_skeleton_label_follows_skeleton_label(
+        self, admin_user, labels, task_id
+    ):
+        new_label_name = "non_skeleton"
+        with make_api_client(admin_user) as api_client:
+            task_labels = [label for label in labels if label.get("task_id") == task_id]
```

Comment:
I would suggest to make sure that there's a skeleton label in there.

Reply:
Added a check